docs/rpc-endpoints.md

```diff
@@ -533,3 +533,8 @@ highest sortition), `reward_cycle` identifies the reward cycle number of this
 tenure, `tip_block_id` identifies the highest-known block in this tenure, and
 `tip_height` identifies that block's height.
 
+### GET /v3/signer/[Signer Pubkey]/[Reward Cycle]
```

Comment:
No need to revert, but I think this file is deprecated.  No need to update it :)